src/main/java/com/eharmony/pho/hbase/PhoenixHBaseDataStoreApiImpl.java

```diff
@@ -28,20 +28,26 @@
 
     public PhoenixHBaseDataStoreApiImpl(final String connectionUrl, final PhoenixHBaseQueryExecutor queryExecutor)
             throws Exception {
+       this(connectionUrl, queryExecutor, false);
+    }
+    
+    public PhoenixHBaseDataStoreApiImpl(final String connectionUrl, final PhoenixHBaseQueryExecutor queryExecutor, final boolean testConnection)
```

Comment:
Just giving the choice to client. By default it was true and i made it as false and if someone want to test the connection before loading the context they can pass the test connection param to the constructor.